syntax = "proto3";

package stream_server;

message RegisterStreamRequest {
  string stream_id = 1;
}

message RegisterStreamResponse {
  bool success = 1;
  string message = 2;
  string stream_id = 3;
}

service StreamServer {
  rpc RegisterStream(RegisterStreamRequest) returns (RegisterStreamResponse);
}
